unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -2065,17 +2131,18 @@ So that the great work for man,
 while he lies in the darkness (for when he is translated into light he will be
 easy) is to distinguish the movings and stirrings of the Spirit of God.
 And this is the best way for man in this state to know them by:
-that which discovers that which is evil, THAT is good.
-That which discovers that which is spiritually evil, THAT must needs be spiritually good;
+that which discovers that which is evil, that is good.
+That which discovers that which is spiritually evil, that must needs be spiritually good;
```

Comment:
Ok, so I might sound super inconsistent, but I think we should either make these capitalized again, or italicize them. My reasoning is that here, these words are capitalized for the sake of _emphasizing_ them, not for pure stylistic reasons (like in sub-headings or when they capitalize the first word of a section).  So, because they were capitalized for the sake of emphasizing the word, we need to retain the emphasis somehow.  Hopefully that distinction makes sense!  I think i slightly favor switching these to italics.

Reply:
Yes, makes total sense.